go/caddy/pfpki/models/models.go

```diff
@@ -759,6 +764,181 @@ func NewCertModel(pfpki *types.Handler) *Cert {
 	return Cert
 }
 
+func (c CA) Resign(params map[string]string) (types.Info, error) {
+	Information := types.Info{}
+	var cadb []CA
+	var err error
+	if val, ok := params["id"]; ok {
+		if err = c.DB.First(&cadb, val).Error; err != nil {
+			Information.Error = err.Error()
+			return Information, err
+		}
+
+	}
+
+	Information.Entries = cadb
+	for _, v := range cadb {
```

Comment:
Why do need the loop?
The logic only handles the first entry of the cadb.
What is happening to the other entries?

Reply:
There is only one entry, i get the ca from the db based on the unique id.